app-crypt/moolticute/moolticute-0.12.9.ebuild

```diff
@@ -23,10 +23,11 @@ IUSE=""
 
 RDEPEND="
 	>=dev-libs/libusb-1.0.20
-	>=dev-qt/qtcore-5.6:5
+	dev-qt/qtcore:5
 	dev-qt/qtgui:5
 	dev-qt/qtnetwork:5
 	dev-qt/qttest:5
+	dev-qt/qttranslations:5
```

Comment:
Where is this dependency coming from?

Reply:
In fact, I notice this bug: https://bugs.gentoo.org/653102 and tried to compile `moolticute` without it, and it doesn't compile.
Now that the `nitrokey-apps` bug is solve, I see that's not the expected dependency, and I wrongly place it into `RDEPEND` instead of `DEPEND`. Will correct that.

Reply:
Fixed